spec/models/user_spec.rb

```diff
@@ -21,8 +37,10 @@
   end
 
   it 'sends an email' do
-    #byebug
-    expect { donor.send_instructions }
-      .to change { ActionMailer::Base.deliveries.count }.by(1)
+   # ActionMailer::Base.deliveries.clear
+    donor.send(:send_registered_email)
```

Comment:
don't use `send` here

instead you should be creating a user in the spec and testing to see that that action sent an email